scss/_global.scss

```diff
@@ -20,35 +20,35 @@ $global-width: rem-calc(1200) !default;
 /// @type Number
 $global-lineheight: 1.5 !default;
 
-/// Colors used for buttons, callouts, links, etc. There must always be a color called `primary`.
+/// Color states used for buttons, callouts, links, etc. There must always be a color called `primary`.
 /// @type Map
-$foundation-palette: (
-  primary: #1779ba,
-  secondary: #767676,
-  success: #3adb76,
-  warning: #ffae00,
-  alert: #cc4b37,
+$foundation-states: (
```

Comment:
Can you elaborate here? Do you mean instead of using the get-color function, we would specify the key that would be used for the `get-color()` function, then return the value using `get-color()` while calling `get-state()`?